### PR TITLE
feat: updating new master with CI/CD config'd

### PR DESCRIPTION
**Pull request by wbrefvem**
This is a replay of #12 against the new master, recently updated to be current with `combined-pipelines`.

### CODE REVIEW

File: jenkins-x.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    value: jenkins-x-bot
+  - name: GOPROXY
+    value: http://jenkins-x-athens-proxy
+  - name: GOOGLE_APPLICATION_CREDENTIALS
+    value: /usr/local/lib/bazel/etc/google-creds.json
+  - name: PROW_REPO_OVERRIDE
+    value: gcr.io/jenkinsxio/prow
+  pipelines:
+    pullRequest:
+      pipeline:
```
Comment (abayer) [inline jenkins-x.yml:15]: It'd probably be good to add `containerOptions` with cpu/memory request/limits like https://github.com/jenkins-x/jx/blob/master/jenkins-x-integration.yml#L27-L35 to the `pipeline`s. Obviously the requests/limits can be higher, but I've found that when we don't specify anything, things can go...poorly.

Comment (wbrefvem) [reply jenkins-x.yml:15]: Done.

File: jenkins-x.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+          image: gcr.io/jenkinsxio/prow-builder:j
+        stages:
+        - name: build
+          steps:
+          - name: build
+            command: bazel
+            args: 
+            - build
+            - //prow/...
+          options:
```
Comment (abayer) [inline jenkins-x.yml:26]: This can actually go on the `pipeline`s, rather than the `stage`s, to simplify things. =)

File: jenkins-x.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ pipelineConfig:
     pullRequest:
       pipeline:
         agent:
-          image: gcr.io/jenkinsxio/prow-builder:0.0.5
+          image: gcr.io/jenkinsxio/prow-builder:j
```
Comment (abayer) [inline jenkins-x.yml]: This looks...wrong. =)

Comment (wbrefvem) [reply jenkins-x.yml]: Yeah idk what that is lol

**Comment by abayer** [comment]
/lgtm cancel
/approve cancel

Whoops, needs `OWNERS` too.

**Pull request merged**